golang/x/relay/client/cli/utils.go

```diff
@@ -3,10 +3,19 @@ package cli
 import (
 	"github.com/spf13/cobra"
 	"io/ioutil"
+	"os"
+	"strings"
 )
 
 func readJSONFromFile(filename string) ([]byte, error) {
-	return ioutil.ReadFile("scripts/json_data/" + filename)
+	// get path to root directory
+	path, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+	// if running this function from cli_test directory do not include it in golang path
+	path = strings.Trim(path, "/cli_test")
```

Comment:
Trim doesn't this way I think. This takes a "cut set" and removes those characters at front and back.  I think you want `TrimSuffix`